src/pages/library/common.js

```diff
@@ -177,6 +179,12 @@ export const SearchAndFilterComponent = ({ fullList, sidebarSections, customSort
       filter: filter || undefined
     }, { addQueryPrefix: true })
 
+    if (filter) {
+      Ajax().Metrics.captureEvent(Events.catalogueFilter + ':search', {
+        filter: filter
+      })
```

Comment:
JavaScript can give you a nice bit of concision here:
```suggestion
      Ajax().Metrics.captureEvent(Events.catalogueFilter + ':search', { filter })
```

Reply:
The entirety of the `if` block could be even more concise 😊 

```javascript
    !!filter && Ajax().Metrics.captureEvent(`${Events.catalogueFilter}:search`, { filter })
```

Reply:
@kyuksel I think one-lining reduces clarity a bit too much

Reply:
`!!something && …` is a pattern we use a lot in this codebase so my brain actually parses it faster that way but I'm fine with leaving it at

```javascript
if (filter) {
  Ajax().Metrics.captureEvent(`${Events.catalogueFilter}:search`, { filter })
}
```

if that reads much better for you.